docs/sources/developers/plugins/share-a-plugin.md

```diff
@@ -0,0 +1,122 @@
+
+Now that you know what Grafana needs to load your plugin, let's see how you can share the plugin with your users.
+
+The best way to share your plugin with the world is to publish it on [Grafana Plugins](https://grafana.com/plugins). However, if you're not ready to make your plugin public just yet, you can still share your plugin by hosting the plugin yourself.
+
+## Publish your plugin on Grafana.com
+
+To publish a plugin to [Grafana Plugins](https://grafana.com/grafana/plugins), your plugin first needs to be publicly available in a commit on [GitHub](https://github.com).
+
+The commit you submit needs to either:
+
+- contain a `dist` directory with the plugin assets
+- contain the plugin assets in the root directory
+
+We strongly recommend that you don't check in the plugin assets to the main branch. Instead, use the following steps to create a release branch that contains the plugin assets.
+
+1. Create a release branch
+
+   ```
+   git checkout -b release-0.1.x
+   ```
+
+1. Build the plugin assets.
+
+   ```
+   yarn build
+   ```
+
+1. Add the `dist` directory. The `-f` flag adds the directory even if it's ignored by `.gitignore`.
+
+   ```
+   git add -f dist
+   ```
+
+1. Create the release commit.
+
+   ```
+   git commit -m "Release v0.1.0"
+   ```
+
+1. Create a release tag. You can also [create the release on GitHub](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository). If you do, you can skip this step, as well as the next one.
+
+   ```
+   git tag -a v0.1.0 -m "Create release tag v0.1.0"
+   ```
+
+1. Push to GitHub. `follow-tags` tells Git to push the release tag along with our release branch.
+
+   ```
+   git push --set-upstream origin release-0.1.x --follow-tags
+   ```
+
+The next step is to submit the URL to your repository, and the release commit, to the [Grafana Plugin Repository](https://github.com/grafana/grafana-plugin-repository).
+
+## Host the plugin yourself
+
+If you want to share your plugin by hosting it yourself, we recommend that you package it by adding the plugin assets to a ZIP archive. You can then make the archive available by hosting them yourself.
+
+How you package the plugin depends on whether you want to include the source code or not.
+
+
+
+### Package the plugin with source code
+
+If you want to distribute the source code along with your plugin assets, you can archive the entire plugin directory.
+
+To create a ZIP archive that contains the plugin assets and source code, run the following commands in your terminal:
+
+```
+cd my-plugin/
+yarn build
+zip my-plugin-0.2.0.src.zip . -r -x "node_modules/*" -x ".git/*"
+```
+
+### Package the plugin without source code
+
+If you don't want to distribute the plugin with the source code, you can instead archive the `dist` directory.
+
+To create a ZIP archive that only contains the bare minimum to load the plugin, run the following commands in your terminal:
```

Comment:
```suggestion
To create a .zip archive that only contains the bare minimum to load the plugin, run the following commands in your terminal:
```

Reply:
Per the MSG, file extensions are all lowercase and preceded by a period. https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions